kymatio/scattering2d/tests/test_torch_scattering2d.py

```diff
@@ -7,66 +7,123 @@
 from kymatio.scattering2d import Scattering2D
 from torch.autograd import gradcheck
 from collections import namedtuple
+
+
+devices = ['cpu']
+if torch.cuda.is_available():
+    devices.append('cuda')
+
+
 backends = []
+backends_devices = []
 
 try:
     if torch.cuda.is_available():
         from skcuda import cublas
         import cupy
         from kymatio.scattering2d.backend.torch_skcuda_backend import backend
         backends.append(backend)
+        if 'cuda' in devices:
+            backends_devices.append((backend, 'cuda'))
 except:
-    pass
+    Warning('torch_skcuda backend not available.')
```

Comment:
Alright this sounds better.

Reply:
Yes and it has the advantage of getting us to :100: test coverage... (For some reason, `pass` always counts as a miss for coverage purposes.)